power_plant_import/tests/data/full_csv.csv

```diff
@@ -668,8 +668,8 @@
 "GD Biopower;GD Geothermal","Plant","Suqian Waste-to-energy Project Phase II","Suqian Waste-to-energy Project Phase II","Suqian Waste-to-energy Project Phase II","Power Plant","China","Northeast Asia",33.96713,118.28871,"Active",,,2017,,,2037,,,,,,70000,"MWh",,,"Geothermal","Geothermal","Industrial Waste","Waste",,,,,"Completed",9,"MW",,,70000,"MWh",,,,,,,,,,,,34554000,"USD",,,,,,,,,2017,"TRUE",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"China Everbright International Ltd",,,,,,,,,,,"1. The plant constructed on a BOT (Build-Operate-Transfer) basis.",,
 "GD Biopower","Project","Suqian Waste-to-energy Project Phase II","Suqian Waste-to-energy Project Phase II","(Project)","Power Plant","China","Northeast Asia",33.96713,118.28871,"Active",,,2017,,,2037,,,,,,70000,"MWh",,,"Geothermal","Geothermal","Industrial Waste","Waste",,,,,"Completed",9,"MW",,,70000,"MWh",,,"Industrial Waste","Waste",,,,,,,,34554000,"USD",,,,,,,,,2017,"TRUE",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"China Everbright International Ltd",,,,,,,,,,,"1. The plant constructed on a BOT (Build-Operate-Transfer) basis.",,
-"GD Biopower;GD Geothermal","Plant","Baoan I Waste to Energy Facility","Baoan I Waste to Energy Facility","Baoan I Waste to Energy Facility","Power Plant","China","Northeast Asia",22.66755,11.87166,"Active",,,2006,,,2026,,,,,,142500,"MWh",,,"Geothermal","Geothermal","Industrial Waste","Waste",,,,,"Completed",19,"MW",,,142500,"MWh",,,,,,,,,,,,,,,,,,,,,,2006,"TRUE",,,,,,,,,,,,,,,,,,,"Shenzhen Energy Environmental Engineering Co Ltd",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
-"GD Biopower","Project","Baoan I Waste to Energy Facility","Baoan I Waste to Energy Facility","(Project)","Power Plant","China","Northeast Asia",22.66755,11.87166,"Active",,,2006,,,2026,,,,,,142500,"MWh",,,"Geothermal","Geothermal","Industrial Waste","Waste",,,,,"Completed",19,"MW",,,142500,"MWh",,,"Industrial Waste","Waste",,,,,,,,,,,,,,,,,,2006,"TRUE",,,,,,,,,,,,,,,,,,,"Shenzhen Energy Environmental Engineering Co Ltd",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+"GD Biopower;GD Geothermal","Plant","Baoan I Waste to Energy Facility","Baoan I Waste to Energy Facility","Baoan I Waste to Energy Facility","Power Plant","China","Northeast Asia",22.824049,113.855187,"Active",,,2006,,,2026,,,,,,142500,"MWh",,,"Geothermal","Geothermal","Industrial Waste","Waste",,,,,"Completed",19,"MW",,,142500,"MWh",,,,,,,,,,,,,,,,,,,,,,2006,"TRUE",,,,,,,,,,,,,,,,,,,"Shenzhen Energy Environmental Engineering Co Ltd",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
Should this one be updated? The screenshot just had Baoan III, this appears to be Baoan I.

Reply:
I'm not sure actually. I was thinking that they should be in the same place, but maybe not. I'll ask on BaseCamp.